src/data.jl

```diff
@@ -1,442 +1,33 @@
-export s18_pha
-const s19_u1 = -5.000000000000000000e-01
-export s19_u1
-const s19_u2 = -8.660254037844383745e-01
-export s19_u2
-const s19_u3 = 0.000000000000000000e00
-export s19_u3
-const s19_v1 = 4.967317648921539819e-01
-export s19_v1
-const s19_v2 = -2.867882181755230797e-01
-export s19_v2
-const s19_v3 = 8.191520442889917986e-01
-export s19_v3
-const s19_per = 4.308204499999999825e04 # periodicity of Sat. 19 [s]
-export s19_per
-const s19_alt = 2.020000000000000000e07 # altitude of Sat. 19 [m]
-export s19_alt
-const s19_pha = 8.712388980384689674e00 # phase of Sat. 19 [rad]
-export s19_pha
-const s20_u1 = 4.999999999999996669e-01
-export s20_u1
-const s20_u2 = -8.660254037844384856e-01
-export s20_u2
-const s20_u3 = 0.000000000000000000e00
-export s20_u3
-const s20_v1 = 4.967317648921540374e-01
-export s20_v1
-const s20_v2 = 2.867882181755229132e-01
-export s20_v2
-const s20_v3 = 8.191520442889917986e-01
-export s20_v3
-const s20_per = 4.308204499999999825e04 # periodicity of Sat. 20 [s]
-export s20_per
-const s20_alt = 2.020000000000000000e07 # altitude of Sat. 20 [m]
-export s20_alt
-const s20_pha = 5.000000000000000000e00 # phase of Sat. 20 [rad]
-export s20_pha
-const s21_u1 = 4.999999999999996669e-01
-export s21_u1
-const s21_u2 = -8.660254037844384856e-01
-export s21_u2
-const s21_u3 = 0.000000000000000000e00
-export s21_u3
-const s21_v1 = 4.967317648921540374e-01
-export s21_v1
-const s21_v2 = 2.867882181755229132e-01
-export s21_v2
-const s21_v3 = 8.191520442889917986e-01
-export s21_v3
-const s21_per = 4.308204499999999825e04 # periodicity of Sat. 21 [s]
-export s21_per
-const s21_alt = 2.020000000000000000e07 # altitude of Sat. 21 [m]
-export s21_alt
-const s21_pha = 6.570796326794896558e00 # phase of Sat. 21 [rad]
-export s21_pha
-const s22_u1 = 4.999999999999996669e-01
-export s22_u1
-const s22_u2 = -8.660254037844384856e-01
-export s22_u2
-const s22_u3 = 0.000000000000000000e00
-export s22_u3
-const s22_v1 = 4.967317648921540374e-01
-export s22_v1
-const s22_v2 = 2.867882181755229132e-01
-export s22_v2
-const s22_v3 = 8.191520442889917986e-01
-export s22_v3
-const s22_per = 4.308204499999999825e04 # periodicity of Sat. 22 [s]
-export s22_per
-const s22_alt = 2.020000000000000000e07 # altitude of Sat. 22 [m]
-export s22_alt
-const s22_pha = 8.141592653589793116e00 # phase of Sat. 22 [rad]
-export s22_pha
-const s23_u1 = 4.999999999999996669e-01
-export s23_u1
-const s23_u2 = -8.660254037844384856e-01
-export s23_u2
-const s23_u3 = 0.000000000000000000e00
-export s23_u3
-const s23_v1 = 4.967317648921540374e-01
-export s23_v1
-const s23_v2 = 2.867882181755229132e-01
-export s23_v2
-const s23_v3 = 8.191520442889917986e-01
-export s23_v3
-const s23_per = 4.308204499999999825e04 # periodicity of Sat. 23 [s]
-export s23_per
-const s23_alt = 2.020000000000000000e07 # altitude of Sat. 23 [m]
-export s23_alt
-const s23_pha = 9.712388980384689674e00 # phase of Sat. 23 [rad]
-export s23_pha
+export DATA_FILE
+const DATA_FILE="data.dat"
```

Comment:
I want this to read from STDIN so we could pass a different file in

Reply:
And also have STDERR printout if the file is not found....
Do this and I'll approve.

Reply:
that's not really how Peter's executables work though, they always read from `data.dat`

Reply:
Also reading a `dat` from STDIN is just not possible since each of these programs take input on stdin

Reply:
Ok that's fine then in that case